pyneuroml/utils/plot.py

```diff
@@ -389,3 +390,184 @@ def load_minimal_morphplottable__model(
                 for acell in inc.cells:
                     acell.biophysical_properties = None
                     nml_model.add(acell)
+
+
+def extract_coordinates_and_diameter(point_with_diam):
+    """
+    Extracts coordinates and diameter from a Point3DWithDiam object.
```

Comment:
For all doc strings, please follow the format already being used in the code. We just use the default reST format (not Google or NumpyDoc). See: https://www.sphinx-doc.org/en/master/usage/domains/python.html#info-field-lists